inyoka_theme_default/static/style/inyoka/portal.less

```diff
@@ -10,6 +10,23 @@
 
 .portal {
 
+  #portal-user-groupselector {
+    list-style: none;
+    padding: 0;
+    max-height: 800px;
```

Comment:
I would like to keep the maximum height cause otherwise the submit button might be quite hard to reach.